src/auto-evo/RunResults.cs

```diff
@@ -182,21 +211,6 @@ public void AddNewSpecies(Species species, IEnumerable<KeyValuePair<Patch, long>
         }
     }
 
-    public void AddSplitResultForSpecies(Species species, Species splitSpecies, List<Patch> patchesToConvert)
-    {
-        if (patchesToConvert == null || patchesToConvert.Count < 1)
-            throw new ArgumentException("split patches is missing", nameof(patchesToConvert));
-
-        MakeSureResultExistsForSpecies(species);
-        MakeSureResultExistsForSpecies(splitSpecies);
-
-        results[species].SplitOff = splitSpecies;
```

Comment:
As this is deleted, how are species now marked as being split?

Reply:
It is done through AddNewSpecies. The population is init through the population sim. .SplitOff probably needs a full rework as it cannot handle two species splitting off of one anyway. AddNewSpecies is how the miche system has done it the whole time, with split result being unused. As far as the gui goes NewlyCreated is set properly so everything displays how it should.

Reply:
I didn't realize that the GUI read the species split from the other side. I guess it would be now then better for clarity to remove the split off property entirely from the species result if it is no longer working / used.